src/styles/index.scss

```diff
@@ -1,3 +1,245 @@
+:root {
+  --light-blue: #00acdc;
+  --column-count: 4;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+// #region header
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+
+  &-logo {
+    display: block;
+  }
+}
+
+.nav {
+  &__list {
+    padding-inline-start: 0;
+    display: inline;
+  }
+
+  &__item {
+    display: inline-block;
+  }
+
+  &__link {
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+    margin-left: 17px;
+    height: 60px;
+    line-height: 60px;
+    display: block;
+  }
+}
+
+.is-active::after {
+  display: inline-block;
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--light-blue);
+  border-radius: 8px;
+}
+
+.is-active {
+  margin-left: 0;
+  height: 60px;
+  position: relative;
+  color: var(--light-blue);
+}
```

Comment:
You have a `margin-left: 0;` property in your `.is-active` class. This will override the `margin-left: 17px;` property in the `.nav__link` class. If you want to remove the margin from the active link, it's better to do it in the `.nav__link.is-active` selector to keep the code cleaner.